From 7be2619feda2c4e28554190261022d3230bfe68a Mon, 10 Oct 2016 16:14:26 +0200
From: Thomas Perl <m@thp.io>
Date: Mon, 1 Jun 2015 12:52:16 +0200
Subject: [PATCH] [zypp] Move dist-upgrade cache to /home/ (Fixes JB#29268)


This helps in situations where the rootfs is low on space.

diff --git a/backends/zypp/pk-backend-zypp.cpp b/backends/zypp/pk-backend-zypp.cpp
index 7bf849d..bc64cd2 100644
--- a/backends/zypp/pk-backend-zypp.cpp
+++ b/backends/zypp/pk-backend-zypp.cpp
@@ -229,7 +229,7 @@
 zypp_set_dist_upgrade_mode (gboolean dist_upgrade_mode)
 {
 	const char *target_path = "/var/cache/pk-zypp-cache";
-	const char *path = dist_upgrade_mode ? "/var/cache/pk-zypp-dist-upgrade"
+	const char *path = dist_upgrade_mode ? "/home/.pk-zypp-dist-upgrade-cache"
 		: "/var/cache/zypp";
 
 	char tmp[PATH_MAX];
